app/assets/javascripts/modules/remove-filter.js

```diff
@@ -33,7 +33,18 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         $input.trigger('change');
       }
       else if (inputType == 'text' || inputType == 'search') {
-        $input.val(currentVal.replace(removeFilterValue, '').replace(/\s+/g,' ').trim()).trigger({
+        var splitKeywords = currentVal.split(" ");
+
+        for (var i = 0; splitKeywords.length; i += 1) {
```

Comment:
Interesting, I'd write this as `i++`, I've not seen it done this way before. Works fine though.

Reply:
Doh, I've got too used to Ruby not having `++` or `--`.